.gitignore

```diff
@@ -31,3 +31,10 @@ launcher/src/version.rc2
 .vs/*
 .vscode
 .idea
+.vagrant/
```

Comment:
Would you like to move some of the gitignores here to the Vagrant + Packer branch?

Reply:
It is my opinion this vranch should be merged with master.  I see no reason to maintain a second branch.